frontend/simple/views/CreateGroup.vue

```diff
@@ -47,11 +47,10 @@
             </button>
           </p>
         </div>
-        <article class="message is-danger" v-if="ephemeral.errorMsg">
-          <div class="message-body">
-            {{ ephemeral.errorMsg }}
-          </div>
-        </article>
+
+        <Message severity='danger' v-if="ephemeral.errorMsg">
```

Comment:
Since all components up to this point have been lowercase, let's maintain that consistency and keep them that way?